asdf/schema.py

```diff
@@ -567,36 +567,6 @@ def get_validator(schema={}, ctx=None, validators=None, url_mapping=None,
     return validator
 
 
-def _validate_large_literals(instance, reading):
-    """
-    Validate that the tree has no large numeric literals.
-    """
-    def _validate(value):
-        # We can count on 52 bits of precision
-        if value <= ((1 << 51) - 1) and value >= -((1 << 51) - 2):
```

Comment:
@perrygreenfield do we still want to limit integers in some way (within range of 64-bit signed, for example)?  Or should we allow trees like this:
```python
with asdf.AsdfFile({"value": 2**70}) as af:
  af.write_to("test.asdf")
```

Reply:
The max bit level check has been added back, limiting an integer to 64 bits.